hw-model/src/model_fpga_realtime.rs

```diff
@@ -0,0 +1,210 @@
+// Licensed under the Apache-2.0 license
+
+use std::{env, str::FromStr};
+
+use caliptra_emu_bus::{Bus, BusError};
+use caliptra_emu_types::{RvAddr, RvData, RvSize};
+use std::io::Write;
+use std::time;
+use uio::{UioDevice, UioError};
+
+use crate::HwModel;
+use crate::Output;
+
+// Static variable to keep track of the handshake with the "uart" code
+static mut TAG: u8 = 1;
+
+// TODO: Make this configurable
+const SOC_PAUSER: u32 = 0xffff_ffff;
+
+fn fmt_uio_error(err: UioError) -> String {
+    format!("{err:?}")
+}
+
+pub struct ModelFpgaRealtime {
+    _dev: UioDevice,
```

Comment:
Is this field being left unused? What is the point of having it in the struct?

Reply:
Removed. I thought this would be RAII but not retaining dev doesn't impact the result.